L/LibMPDec/bundled/patches/01-libname-windows.patch

```diff
@@ -1,15 +1,18 @@
 --- a/configure.ac
 +++ b/configure.ac
-@@ -219,6 +219,12 @@
-         LINK_DYNAMIC="-Wl,-bshared"
-         LINK_SHLIBS="../libmpdec/$LIBIMPORT"
-         ;;
-+    MSYS_NT*)
-+        LIBNAME="libmpdec.dll"
-+        LIBSONAME="libmpdec-3.dll"
-+        LIBSHARED="libmpdec-2.5.1.dll"
-+        CONFIGURE_LDFLAGS="-shared"
-+        ;;
-     *)
-         LIBNAME="libmpdec.so"
-         LIBSONAME="libmpdec.so.3"
+@@ -359,11 +359,11 @@
+     ;;
+   *-*-mingw*)
+     FPIC=
+-    LIBNAME=
++    LIBNAME="libmpdec.dll"
+     LIBIMPORT="libmpdec.dll.a"
+-    LIBSONAME=
+-    LIBSHARED="libmpdec-4.dll"
+-    CONFIGURE_LDFLAGS="-shared -Wl,--out-implib,$LIBIMPORT"
++    LIBSONAME="libmpdec-4.dll"
++    LIBSHARED="libmpdec-4.0.0.dll"
```

Comment:
Why this?  Doesn't this mean that all versions are ABI breaking?  that sounds very unideal.

Reply:
That's what the previous patch did. I only tried to update it to the new configure file. However, this updated patch is not working (and is not used). I forgot to delete it, and the respective `#TODO` lines in the build script.